src/realm/util/serializer.cpp

```diff
@@ -171,6 +171,16 @@ std::string print_value<>(realm::UUID uuid)
     return "uuid(" + uuid.to_string() + ")";
 }
 
+StringData get_printable_table_name(StringData name)
```

Comment:
There's `ObjectStore::object_type_for_table_name` which does the same thing - would it make sense to unify them somehow, now that everything is in the same repo?

Reply:
It's a good point, we should do that at some point. But this version is a bit more permissive, and I don't mind the small duplication until we can spend some time on figuring out the dependencies between object-store and core a bit more.